lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.html

```diff
@@ -0,0 +1,32 @@
+<adf-search-filter-tabbed
+    [settings]="settings"
+    [queries]="queries"
+    [valuesToDisplay]="valuesToDisplay"
+    (fieldsChanged)="onFieldsChanged($event)"
+    (displayedLabelsByFieldTranslated)="onDisplayLabelsByFieldsTranslated($event)"
+    (queriesCombined)="onQueriesCombined($event)"
+    (valuesToDisplayCombined)="onValuesToDisplayCombined($event)">
+    <ng-container *ngFor="let field of fields">
```

Comment:
I don't think we need ngFor here, date range will not have more than 2 fields and usage of ngFor adds the need to provide `displayedLabelsByField` setting and whole logic to translate those label in `SearchFilterTabbedComponent`. Using `adf-search-date-range-advanced` twice directly in HTML will allow us to delete this setting and whole logic to translate it making the code more readable. @AleksanderSklorz what do you think?

Reply:
fine for me :)

Reply:
We do need an *ngFor here, since its is not necessary for the tab to have 2 fields. The Dublin Core filtering set, for e.g., only allows the Created Date filter, and not the Modified Date filter
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/92505353/b1dfa357-7232-49d0-b69a-e4c81268ce0e)